crates/ty_python_semantic/resources/mdtest/snapshots/super.md_-_Super_-_Basic_Usage_-_Explicit_Super_Objec…_(b753048091f275c0).snap

```diff
@@ -128,6 +128,7 @@ error[unresolved-attribute]: Type `<super: <class 'C'>, C>` has no attribute `c`
 20 |
 21 | super(B, C()).a
    |
+info: Python 3.12 was assumed when accessing `c` because it was specified on the command line
 info: rule `unresolved-attribute` is enabled by default
```

Comment:
These diagnostics for accesses to `super()` are very sad and maybe we should special case them? I'm not sure if there's a more general rule that excludes super and similar objects.